cmd/v2upgrade.go

```diff
@@ -0,0 +1,90 @@
+package cmd
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/chanzuckerberg/fogg/config"
+	v1 "github.com/chanzuckerberg/fogg/config/v1"
+	"github.com/chanzuckerberg/fogg/errs"
+	"github.com/chanzuckerberg/fogg/plan"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	v2upgrade.Flags().StringP("config", "c", "fogg.json", "Use this to override the fogg config file.")
+	rootCmd.AddCommand(v2upgrade)
+}
+
+var v2upgrade = &cobra.Command{
+	Use:   "v2upgrade",
+	Short: "Upgrades a v1 config to a v2 config",
+	Long: `This command will upgrade a v1 config to a v2 config.
+	Note that this is a lossy transformation.`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		var err error
+		// Set up fs
+		pwd, err := os.Getwd()
+		if err != nil {
+			return errs.WrapUser(err, "can't get pwd")
+		}
+		fs := afero.NewBasePathFs(afero.NewOsFs(), pwd)
+
+		// handle flags
+		configFile, err := cmd.Flags().GetString("config")
+		if err != nil {
+			return errs.WrapInternal(err, "couldn't parse config flag")
+		}
+
+		// check that we are at root of initialized git repo
+		openGitOrExit(fs)
+
+		bytes, version, err := config.FindConfig(fs, configFile)
+		if err != nil {
+			return err
+		}
+		switch version {
```

Comment:
I think this core logic probably should be moved to the config package. That should make it easier to test.